crates/bevy_ecs/src/world/entity_ref.rs

```diff
@@ -1688,7 +1688,6 @@ impl<'w> EntityWorldMut<'w> {
             })
         };
 
-        #[allow(clippy::undocumented_unsafe_blocks)] // TODO: document why this is safe
```

Comment:
Yes, this means clippy will complain about an undocumented unsafe block.

Unfortunately, I have *no* idea what the safety doc should be, or if it'd even be correct.